src/clp_ffi_js/ir/IrStreamReader.hpp

```diff
@@ -0,0 +1,113 @@
+#ifndef CLP_FFI_JS_IR_IR_STREAM_READER_HPP
```

Comment:
I'll add a linter for this soon, but the format of header defines should be `<NAMESPACE>_<FILENAME>_HPP`, which in this case would be `CLP_FFI_JS_IR_IRSTREAMREADER_HPP`.